content/agent/proxy.md

```diff
@@ -144,56 +146,74 @@ Once the HAProxy configuration is in place, you can reload it or restart HAProxy
 
-Then edit each Agent to point to HAProxy by setting its `dd_url` to the address of HAProxy (e.g. haproxy.example.com). This `dd_url` setting can be found in `datadog.conf` for Agent v5 and `datadog.yaml` for Agent v6.
+### Datadog Agent configuration
+#### Agent v>6
+
+Edit each Agent to point to HAProxy by setting its `dd_url` to the address of HAProxy (e.g. `haproxy.example.com`). 
+This `dd_url` setting can be found in the `datadog.yaml` file. 
 
 `dd_url: https://haproxy.example.com:3834`
 
-If you want to send traces or processes through the proxy, you need to setup the following in `datadog.conf` for Agent v5:
+If you want to send traces or processes through the proxy, you must setup the following in the `datadog.yaml` file:
 
 ```
-[trace.api]
-endpoint = https://haproxy.example.com:3835
+apm_config:
+    endpoint: https://haproxy.example.com:3835
 
-[process.api]
-url = https://haproxy.example.com:3836
+process_config:
+    url: https://haproxy.example.com:3836
 ```
 
-For Agent v6, set up the following in `datadog.yaml`:
+Then edit the `datadog.yaml` Agent configuration file and set `skip_ssl_validation` to `true`. This is needed to prevent python from complaining about the discrepancy between the hostname on the SSL certificate (`app.datadoghq.com`) and your HAProxy hostname.:
 
 ```
-apm_config:
-    endpoint: https://haproxy.example.com:3835
+skip_ssl_validation: true
+```
+
+Finally [restart the Agent][4].
+
+To verify that everything is working properly, review the HAProxy statistics at `http://haproxy.example.com:3835` as well as the [Infrastructure Overview][5]
+
+#### Agent v<6
+
+Edit each Agent to point to HAProxy by setting its `dd_url` to the address of HAProxy (e.g. `haproxy.example.com`). 
+This `dd_url` setting can be found in the `datadog.conf` file. 
+
+`dd_url: https://haproxy.example.com:3834`
+
+If you want to send traces or processes through the proxy, you must setup the following in the `datadog.conf` file: 
 
-process_config:
-    url: https://haproxy.example.com:3836
 ```
+[trace.api]
+endpoint = https://haproxy.example.com:3835
 
-Before you [restart the Agent][2] Edit your supervisor configuration to disable SSL certificate verification. This is needed to prevent python from complaining about the discrepancy between the hostname on the SSL certificate (app.datadoghq.com) and your HAProxy hostname.
+[process.api]
+url = https://haproxy.example.com:3836
+```
 
-####  On GNU/Linux, Mac OS X, FreeBSD, SmartOS:
-You need to edit the supervisor configuration found at:
+Edit your supervisor configuration to disable SSL certificate verification. This is needed to prevent python from complaining about the discrepancy between the hostname on the SSL certificate (`app.datadoghq.com`) and your HAProxy hostname. The supervisor configuration found at:
 
 * `/etc/dd-agent/supervisor_ddagent.conf` on debian-based systems
 * `/etc/dd-agent/supervisor.conf` on redhat-based systems
```

Comment:
should Debian and Red Hat be capitalized?

Reply:
yes